dhcpv6/nclient6/client.go

```diff
@@ -234,12 +234,21 @@ func WithBroadcastAddr(n *net.UDPAddr) ClientOpt {
 // Matcher matches DHCP packets.
 type Matcher func(*dhcpv6.Message) bool
 
-// IsMessageType returns a matcher that checks for the message type.
+// IsMessageType returns a matcher that checks for one of several message
+// types.  It is most efficient to put the most likely type first.
 //
 // If t is MessageTypeNone, all packets are matched.
-func IsMessageType(t dhcpv6.MessageType) Matcher {
+func IsMessageType(t dhcpv6.MessageType, tt ...dhcpv6.MessageType) Matcher {
```

Comment:
Why not just make it a vararg, instead of a separate first arg + vararg?

Reply:
This syntax enforces the presence of at least one argument at compile time, without resorting to error handing which would conplicate that API